components/news_summary_process.py

```
@@ -0,0 +1,120 @@
+    summary_min, summary_max = decide_summary_len(lang, st.session_state['summary_len']).values()
+    with st.spinner("생성 중입니다..."):
+        try:
+            summary = summary_news(text, lang, summary_min, summary_max)
+        except IndexError:
+            st.error("입력 길이를 다시 확인해주세요. (권장 길이 100 ~ 2000자)")
+            return
+        title = generate_title(summary, lang)
+        topic = detect_topic(text if len(text) <= 512 else summary)
+        try:
+            save_article(title, summary, topic, lang)
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)